meta-mbl-apps/recipes-devtools/mbed-cli/mbed-cli_1.10.0.bb

```
@@ -5,6 +5,7 @@
 LICENSE="Apache-2.0"
 LIC_FILES_CHKSUM = "file://LICENSE;md5=4336ad26bb93846e47581adc44c4514d"
 
+DEPENDS = "mercurial-native"
```

I don't think the `-native` bit should be here. mbed-cli depends on `mercurial-native` when it's being built for the host machine, but depends on `mercurial` when it is being built for the target. That's the behaviour we'd get by writing
```
DEPENDS = "mercurial"
```
because when building `-native` recipes, BitBake will automatically convert recipes in `DEPENDS` to be `-native`.

...Shouldn't this be an `RDEPENDS` rather than  a `DEPENDS`? Do we need mercurial to be around when we build mbed-cli or just when it's being run?

```suggestion
RDEPENDS_${PN} += "mercurial"
```

---

Both points make sense. I wasn't aware of the automatic conversion. The RDEPENDS is correct, because mercurial is required runtime. The same applies to git as well, since both are used by the mbed-cli, so in theory it could be added as well.

---

Thanks for the changes.